app/models/site_status.rb

```diff
@@ -100,7 +100,7 @@ def self.default_vac_status_given(study_mode:)
   end
 
   def has_vacancies?
-    SiteStatus.findable.with_vacancies.any?
+    with_vacancies?
```

Comment:
It would be faster to keep the `any?` on here and we're expecting it to return a boolean.

```
2020-02-26 09:42:07.047078 D [56616:70222132461600] (0.407ms) ActiveRecord -- SiteStatus Exists? -- { :sql => "SELECT 1 AS one FROM \"course_site\" WHERE \"course_site\".\"vac_status\" != $1 LIMIT $2", :binds => { :vac_status => "", :limit => 1 } }
2020-02-26 09:42:24.551091 D [56616:70222132461600] (42.8ms) ActiveRecord -- SiteStatus Load -- { :sql => "SELECT \"course_site\".* FROM \"course_site\" WHERE \"course_site\".\"vac_status\" != $1", :binds => { :vac_status => "" } }
```

The difference probably wouldn't be as dramatic here as it's a smaller result set that would be returned but every little bit helps 😄

Reply:
`with_vacancies?` returns a boolean already so i'm not sure I can use `.any?` :thinking:

Reply:
The old method was incorrect in that it operated on a result set rather than a single instance

Reply:
Ah yes. So it does. Misread it 👍